test/fixtures/generator/bwb.xcodeproj/project.pbxproj

```diff
@@ -4215,6 +4215,7 @@
 				BAZEL_LABEL = "@//tools/generators/legacy/test:tests";
 				BAZEL_OUTPUTS_DSYM = "\"bazel-out/applebin_macos-darwin_x86_64-opt-STABLE-4/bin/tools/generators/legacy/test/tests.xctest.dSYM\"";
 				BAZEL_OUTPUTS_PRODUCT = "bazel-out/applebin_macos-darwin_x86_64-opt-STABLE-4/bin/tools/generators/legacy/test/tests.__internal__.__test_bundle.zip";
+				BAZEL_OUTPUTS_PRODUCT_BASENAME = tests.xctest;
```

Comment:
This change shows how the old way would be `tests.__internal__.__test_bundle.zip` but now is `tests.xctest`.